src/angular/hq/src/app/clients/client-details/client-details-summary/client-details-summary.component.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <h1 class="font-rajdhani font-semibold text-3xl">{{client?.name}}</h1>
+
+  <div class="grid grid-flow-col gap-[30px] auto-cols-max pt-3">
+      <div class="text-gray-50">hourly rate: ${{client?.hourlyRate}}</div>
+      <div class="text-gray-50">email: {{client?.billingEmail}}</div>
+      <div class="text-gray-50">official name: {{client?.officialName}}</div>
+  </div>
+  @if(this.apiErrors) {
+    @for (error of this.apiErrors; track $index) {
+    <div class="text-red-700 text-xs leading-normal inline-block">{{ error }}</div>
+    }
+    }
```

Comment:
Could we make a ErrorDisplay component in the common components that does this logic? Add an `@Input` for the `string[]`. Update the other forms/pages that display the API errors to use the new component, so it should end up looking something like this after the changes:

```
<hq-error-display [apiErrors]="apiErrors"></hq-error-display>
```